src/cloud-formation/deploy-indexer-main.yaml

```diff
@@ -1,26 +1,26 @@
 ---
 parameters:
-  BrokerImageVersion: '1.1.0'
-  DeployAlb: 'false'
-  DeployAlbDnsRecord: 'false'
-  DeployBastionHost: 'false'
-  DeployBroker: 'false'
-  DeployContainers: 'false'
-  DeployDb: 'false'
-  DeployNlb: 'false'
-  DeployNlbVpcLink: 'false'
-  DeployPostgrest: 'false'
-  DeployProcessor: 'false'
-  DeployRestApi: 'false'
-  DeployRestApiDnsRecord: 'false'
-  DeployStack: 'false'
+  BrokerImageVersion: '2.0.0'
```

Comment:
- [ ] resolve this only once the stack has been torn down and is ready for full re-apply

Reply:
Just updated 30 seconds ago! 
![image](https://github.com/user-attachments/assets/4a29de74-191f-4c9a-861d-6b067090ced5)